### PR TITLE
osdocs636 default GCP install

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://jira.coreos.com/browse/OSDOCS-636

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@abhinavdahiya, @jstuever, will you PTAL at these draft GCP docs? If this looks ok to you, I'll send it over to QE.

**Review by jstuever — CHANGES_REQUESTED** [review]
It appears `installing-gcp-customizations.html` refers to aws in `Installation configuration parameters`. Other than that, just a few minor issues.

**Comment by kalexand-rh** [comment]
@openshift/team-documentation PTAL

File: modules/installation-configuration-parameters.adoc [hunk 1]
```diff
@@ -102,6 +113,32 @@ accounts for the dramatically decreased machine performance.
 ====
 |`Enabled` or `Disabled`
 
+|`compute.replicas`
+|The number of compute, or worker, machines to provision.
+|A positive integer greater than or equal to `2`. The default value is `3`.
+
+|`controlPlane.hyperthreading`
+|Whether to enable or disable simultaneous multithreading, or `hyperthreading`
```
Comment (bobfuru) [inline modules/installation-configuration-parameters.adoc]: s/`hyperthreading` on/`hyperthreading`, on

File: modules/installation-gcp-config-yaml.adoc [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+sequence of mappings. To meet the requirements of the different data structures,
+the first line of the `compute` section must begin with a hyphen, `-`, and the
+first line of the `controlPlane` section must not. Although both sections
+currently define a single machine pool, it is possible that future versions
+of {product-title} will support defining multiple compute pools during
+installation. Only one control plane pool is used.
+<4> Whether to enable or disable simultaneous multithreading, or
+`hyperthreading`. By default, simultaneous multithreading is enabled
+to increase the performance of your machines' cores. You can disable it by
+setting the parameter value to `Disabled`. If you disable simultanous
```
Comment (bobfuru) [inline modules/installation-gcp-config-yaml.adoc]: s/simultanous/simultaneous

File: modules/installation-gcp-config-yaml.adoc [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+<5> To configure faster storage for etcd, especially for larger clusters, set the
+storage type as `io1` and set `iops` to `2000`.
+<6> You can optionally provide the `sshKey` value that you use to access the
+machines in your cluster.
++
+[NOTE]
+====
+For production {product-title} clusters on which you want to perform installation
+debugging or disaster recovery, you must provide an SSH key that your `ssh-agent`
+process uses to the installation program.
```
Comment (bobfuru) [inline modules/installation-gcp-config-yaml.adoc]: s/to the/to access the

File: modules/installation-launching-installer.adoc [hunk 4]
```diff
@@ -37,6 +47,16 @@ cluster.
 
 .Procedure
 
+ifdef::gcp[]
+. Remove any existing GCP credentials that do not use the service account key
+for the GCP account that you configured for your cluster and are stored in the
```
Comment (bobfuru) [inline modules/installation-launching-installer.adoc]: s/and are/and that are

**Review by bobfuru — COMMENTED** [review]
A few suggestions, similar to https://github.com/openshift/openshift-docs/pull/16374/

**Comment by kalexand-rh** [comment]
Thanks @bobfuru!

**Comment by vikram-redhat** [comment]
@wmengRH is the default GCP install content ok from QE's point of view?

**Comment by wmengRH** [comment]
Yes, the default GCP install content LGTM.

**Pull request merged**